website/docs/r/cloudwatch_log_subscription_filter.html.markdown

```diff
@@ -37,3 +37,11 @@ The following arguments are supported:
 ## Attributes Reference
 
 No extra attributes are exported.
+
+## Import
+
+CloudWatch Logs subscription filter can be imported using the log group name and subscription filter name separated by `|`.
```

Comment:
Nit: We really need to document and start standardizing on a single import separator. 😅 Preferably this would be something more like a `,` comma to not interfere with the shell `|` pipe operator, but not going to stop this pull request over this as it is something we should address holistically across the provider. (Maybe commas won't be right either.)